requirements_dev.txt

```diff
@@ -1,6 +1,7 @@
 pytest
 flake8
 pytest-flake8
+requests-mock
```

Comment:
I didn't know this package existed. I wonder if we can apply to our notebooks to make truly offline tests.